packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -478,14 +504,118 @@ describe('plugin-meetings', () => {
         );
       });
 
-      it('emits the correct frameRate', async () => {
-        await startStatsAnalyzer({expected: {receiveVideo: true}});
+      it('emits the correct transmittedFrameRate/receivedFrameRate', async () => {
```

Comment:
each of these tests are absolutely ginormous - could we break it up into smaller component parts and nest the its? does that make sense?

Reply:
Yep, will see what I can do to better organize them.